src/components/sidebar/SidebarHeader/index.tsx

```diff
@@ -96,12 +96,22 @@ const SafeHeader = (): ReactElement => {
 
           <Track {...OVERVIEW_EVENTS.COPY_ADDRESS}>
             <CopyButton text={addressCopyText} className={css.iconButton}>
-              <SvgIcon component={CopyIconBold} inheritViewBox color="primary" fontSize="small" />
+              <SvgIcon
+                component={CopyIconBold}
+                inheritViewBox
+                color="primary"
+                fontSize="small"
+                sx={{
+                  '& path': {
+                    fill: ({ palette }) => palette.primary.main,
+                  },
+                }}
```

Comment:
If you add `class="illustration-main-fill"` to the `copy-boly.svg`, it achieves the same.

You have to keep the `fill` attribute otherwhise this happens for some reason:

![image](https://user-images.githubusercontent.com/20442784/204484866-93f7d0ad-1f4a-4fa1-9f6f-a3cb905e090f.png)

Reply:
Added fill="currentColor" as per Usame's comment.